CDCgov/prime-reportstream
# prime cli fake input generation

This PR ...

Test Steps:
1. Run command `./prime data --input-fake 50 --input-schema csv-otc-covid-19 --output PATH/TO/pdi-covid-19-ri.csv --target-states RI`
2. Open the output file.
3. Verify that the the produced csv file contains only one patient_state column, that it is RHODE ISLAND.
4. Verify that the patient_county column is mostly actual counties with the occasional random string.


## Changes
- Changed CsvSerializer.kt to only include one (1) of each named csv field and to prefer non-TEXT types when available.
- Changed FakeReport.kt to properly choose target state from a valueset
## Checklist

### Testing

## Linked Issues
- Fixes #7897
